core/extensions.rs

```diff
@@ -379,6 +404,11 @@ macro_rules! extension {
       }
 
       #[allow(dead_code)]
+      /// Initialize this extension's OPs
```

Comment:
```suggestion
      /// Initialize this extension for runtime or snapshot creation, excluding
      /// its JavaScript sources and evaluation. This is used when the runtime
      /// or snapshot is created from a (separate) snapshot which includes this
      /// extension in order to avoid evaluating the JavaScript twice.
```